### PR TITLE
Add slurp service

### PR DESCRIPTION
**Pull request by i-like-robots**
Adds basic Slurp service.

Only one endpoint added for now to generate the dynamic related nav consumed by dobi.

### CODE REVIEW

**Comment by matthew-andrews** [comment]
lgtm :+1:

**Comment by matthew-andrews** [comment]
althoguh……

**Comment by matthew-andrews** [comment]
typically next-ft-api-client doesn't edit any of the responses from its upstream apis

File: src/slurp.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+			return response;
+		})
+		.then(fetchres.json);
+}
+
+exports.cooccurancesAsCounts = function(id, types) {
+	return slurp('/cooccurrences_as_counts/' + id).then(function(response) {
+		var tags = response.cooccurrences_as_counts;
+
+		return tags
```
Comment (matthew-andrews) [inline src/slurp.js]: this ought not to be in next-ft-api-client :see_no_evil:

Comment (wheresrhys) [inline src/slurp.js]: @railsagainstignorance Are you open to change requests

File: src/slurp.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+			.map(function(tag) {
+				tag = tag[0].split(':');
```
Comment (commuterjoy) [inline src/slurp.js]: redefining a variable is a bit confusing. doesn't jshint moan about this?

Comment (matthew-andrews) [inline src/slurp.js]: you think so? i do this a lot. nice and functional……

File: src/slurp.js (hunk 1 above)
Comment (i-like-robots) [inline src/slurp.js]: moved!

**Comment by matthew-andrews** [comment]
thank you!

**Comment by matthew-andrews** [comment]
:+1:

**Pull request merged**